pkg/ccl/backupccl/backup_test.go

```diff
@@ -8703,3 +8703,60 @@ DROP TABLE foo;
 	defer cleanupEmptyCluster()
 	sqlDBRestore.Exec(t, "RESTORE FROM $1 AS OF SYSTEM TIME "+aost, LocalFoo)
 }
+
+// TestRestoreNewDatabaseName tests the new_db_name optional feature for single database
```

Comment:
nice test cleanup. Some formatting still looks off so I think the linter will shout at you.